spec/models/volunteer_op_spec.rb

```diff
@@ -117,4 +117,53 @@
       expect(VolunteerOp.search_for_text('good')).to eq([vol_op2])
     end
   end
+  
+  describe '#full_address' do
+    let(:details) do
+      {
+        title: 'test',
+        description: 'description',
+        address: 'Station Rd',
+        postcode: 'HA8 7BD',
+        organisation_id: 1
+      } 
+    end
+    let!(:vol_op) { FactoryGirl.create :volunteer_op, details }
+    
+    it 'returns a full address' do
+      expect(vol_op.full_address).to eq 'Station Rd, HA8 7BD'
+    end
+  end
+  
+  describe 'set\'s volunteer_op lat and lng' do
+    let(:details) do
+      {
+        title: 'test',
+        description: 'description', 
+        address: 'Station Rd',
+        postcode: 'HA8 7BD',
+        organisation_id: 1
+      } 
+    end
+    let!(:vol_op) { FactoryGirl.create :volunteer_op, details }   
+    
+    it 'has a different address' do
```

Comment:
I suggest to extract the test of `address_complete?` method to it's own describe block